Source/ASPagerNode.m

```diff
@@ -112,7 +113,15 @@ - (void)didLoad
 
 - (NSInteger)currentPageIndex
 {
-  return (self.view.contentOffset.x / CGRectGetWidth(self.view.bounds));
+  return (self.view.contentOffset.x / [self pageSize].width);
```

Comment:
The old implementation works as expected, but let's not assume that `self.view.bounds.width == self.pageSize.width`.